presto-main/src/main/java/com/facebook/presto/sql/planner/LiteralInterpreter.java

```diff
@@ -52,7 +52,6 @@
 import io.airlift.slice.Slice;
 import io.airlift.slice.SliceOutput;
 import io.airlift.slice.SliceUtf8;
-import io.airlift.slice.Slices;
```

Comment:
> type is asserted earlier.
What do you mean? Where is it asserted?

Reply:
There is a line:
```

Reply:
There is a line:
```
checkArgument(Primitives.wrap(type.getJavaType()).isInstance(object), ..
```

so no point to check `object instanceof String` when `type instanceof VarcharType`. `VarcharType` is represented by a `Slice`.